daphne/src/testing.rs

```diff
@@ -61,23 +67,31 @@ pub const COLLECTOR_BEARER_TOKEN: &str = "syfRfvcvNFF5MJk4Y-B7xjRIqD_iNzhaaEB9mY
 pub(crate) struct MockAggregator {
     tasks: HashMap<Id, DapTaskConfig>,
     hpke_receiver_config_list: Vec<HpkeReceiverConfig>,
+    pub report_store: Arc<Mutex<HashMap<BucketInfo, ReportStore>>>,
```

Comment:
This is because we have a unit test that directly edits the `collected` field in the report_store.
We can remove this `pub` if we remove that unit test.
(See [here](https://github.com/nakatsuka-y/daphne/blob/11b9205ff349ba7729c58dc6ed797d0e3b530949/daphne/src/roles_test.rs#L248))

Reply:
Because the test is in the same crate as this struct, we can make this `pub(crate)` instead of `pub`. This is better because it ensures that, once we make `MockAggregator` `pub` instead of `pub(crate)`, this field doesn't get exposed in the public API.

Reply:
Ah, I understood your comment now. I wasn't sure what the difference between `pub` and `pub(crate)` was, but I think I got the idea.